src/tests/mod.rs

```diff
@@ -99,6 +99,7 @@ macro_rules! generate_store_tests {
         glue!(ordering, ordering::ordering);
         glue!(order_by, order_by::order_by);
         glue!(sql_types, data_type::sql_types::sql_types);
+        glue!(numeric, data_type::numeric::numeric);
```

Comment:
Current `numeric` test looks that it only covers `INT(8)` data type, how about using `int8`?